data/geoscience/alluvial-fans/index.html

```diff
@@ -0,0 +1,54 @@
+  <p class="caption-text">Alluvial Fans</p>
+</div>
+{% include abstract.html download="???????????"
+name="Alluvial Fans"
+stewards="Utah Division of Emergency Management (DEM)"
+abstract="Active Alluvial Fan designations for Utah. Data represents a combination of existing geologic mapping 1964 – 2013, USGS Quadrangles 1:24,000, 2-ft contour data, historical and modern aerial photography 1937 – 2017, UGS flood and landslide hazard mapping, FEMA National Flood Hazard Layer, and field investigation with photos. The primary risk classification used are high and medium based on the presence of population at risk. Data is supported by a full study report. More detailed information on alluvial fan hazards in Utah is available from the Division of Emergency Management." %}
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="AlluvialFans">
+      <a href="#AlluvialFans" class="anchor--link">Alluvial Fans</a>
+    </h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <p>Alluvial fan landforms pose unique flooding hazards that are not present on non-fan landforms (e.g. rivers, streams, whose flooding hazards are commonly defined through Federal Emergency Management Agency (FEMA) regulatory floodplains). FEMA defines alluvial fan flooding as “Flooding occurring on the surface of an alluvial fan or similar landform which originates at the apex and is characterized by high-velocity flows; active processes of erosion, sediment transport, and deposition; and unpredictable flowpaths. Alluvial fan flooding is depicted on a Flood Insurance Rate Map (FIRM) as Zone AO, with a flood depth and velocity” (FEMA, 2003). Alluvial fans are generally located at a topographic apex between a mountain front and the adjacent piedmont landform. Properly identifying alluvial fan landforms is a critical first step in understanding flood risk to downstream development.</p>
+
+      <p>Active alluvial fan landforms are defined by location, composition, and flooding characteristics. They are located at a topographic break between a mountain front and the piedmont, they are composed of unconsolidated alluvium, and exhibit flooding characteristics described previously. Inactive alluvial fan landforms are geologically older and are no longer subject to active fan flooding processes. As they age, inactive alluvial fans develop their own internal, tributary drainage networks that exhibit riverine flooding characteristics.</p>
+
+      <p>It is recommended that the alluvial fan landform GIS layer and supporting data be used by regulatory officials (federal, state, local, and tribal) when evaluating development and flooding hazards in their jurisdictions.</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://gis.utah.gov/downloads/Active Alluvial Fan Landform Inventory - Wasatch and Oquirrh Ranges FINAL.zip">Active Alluvial Fan Landform Inventory Report- Wasatch and Oquirrh Ranges</a>
```

Comment:
please do not use gis.utah.gov links. add the {{ style links. Also, I would recommend renaming this file to

1. Not have `FINAL` in the name
1. Remove the spaces in the file name.